### PR TITLE
[D1] Adding info on `wrangler d1 insights`

### PR DESCRIPTION
**Pull request by Oxyjun**
### Summary

1. Documenting the full list of available options in a table, as of Wrangler 3.95.0
2. Adding in actual responses from the example `d1 insight` commands.
3. Adding a note to define/explain `queryEfficiency`.
4. Updating examples to use `--limit` rather than the outdated `--count`.

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

File: src/content/docs/d1/observability/metrics-analytics.mdx [hunk 1]
```diff
@@ -152,22 +154,173 @@ Run `wrangler d1 insights --help` to view current options.
 
 :::
 
-### Examples
+| Option             | Description                                                                                                      |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------- |
+| `--timePeriod`     | Fetch data from now to the provided time period (default: `1d`).                                                 |
+| `--sort-type`      | The operation you want to sort insights by. Select between `sum` and `avg` (default: `sum`).                     |
+| `sort-by`          | The field you want to sort insights by. Select between `time`, `reads`, `writes`, and `count` (default: `time`). |
```
Comment (marciocloudflare) [inline src/content/docs/d1/observability/metrics-analytics.mdx]: Is this the only command without `--` at the beginning?

File: src/content/docs/d1/observability/metrics-analytics.mdx [hunk 2]
```diff
@@ -152,22 +154,173 @@ Run `wrangler d1 insights --help` to view current options.
...
+    "totalDurationMs": 0.9901,
+    "numberOfTimesRun": 2,
+    "queryEfficiency": 0
+  }
+]
+```
+</Details>
+
+:::note
+The quantity `queryEfficiency` measures how efficient your query was. It is calculated as "the number of rows returned, divided by the number of rows read".
```
Comment (marciocloudflare) [inline src/content/docs/d1/observability/metrics-analytics.mdx]:
```suggestion
The quantity `queryEfficiency` measures how efficient your query was. It is calculated as the number of rows returned, divided by the number of rows read.
```

Comment (marciocloudflare) [reply src/content/docs/d1/observability/metrics-analytics.mdx]: Avoid having sentences in between quotes.

**Review by marciocloudflare — APPROVED** [review]

File: src/content/docs/d1/observability/metrics-analytics.mdx (hunk 1 above)
Comment (Oxyjun) [reply src/content/docs/d1/observability/metrics-analytics.mdx]: Good catch! Thanks.

**Comment by Classicrichblog** [comment]
> ### Summary
> 
> 
> 
> 
> 
> 
> 
> 1. Documenting the full list of available options in a table, as of Wrangler 3.95.0
> 
> 2. Adding in actual responses from the example `d1 insight` commands.
> 
> 3. Adding a note to define/explain `queryEfficiency`.
> 
> 4. Updating examples to use `--limit` rather than the outdated `--count`.
> 
> 
> 
> ### Screenshots (optional)
> 
> 
> 
> 
> 
> 
> 
> ### Documentation checklist
> 
> 
> 
> 
> 
> 
> 
> - [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
> 
> - [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
> 
> - [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).
> 
>

**Comment by vy-ton** [comment]
@rozenmd could you review here too

**Review by rozenmd — APPROVED** [review]

**Pull request merged**